implementation/testHelpers/helpers/assertBalance.js

```diff
@@ -2,6 +2,7 @@ const BN = require('bn.js')
 const chai = require('chai')
 const expect = chai.expect
 const bnChai = require('bn-chai')
+const {web3} = require("@openzeppelin/test-environment")
```

Comment:
Did this get used anywhere?

Reply:
used in `increaseTime()`

Reply:
That's not called in this file, right?

Reply:
no, but it's called in some test files

Reply:
… I guess I'm asking, why are we importing it in this file?

Reply:
Tests using `increaseTime` fail with `web3 is not defined' error